web/src/Components/Bridging/BridgeContainer.tsx

```diff
@@ -20,11 +21,11 @@ type BridgeContainerProps = {
 export default ({ bridges, mode, setMode, bridgeValue, setValue }: BridgeContainerProps) => {
   const [sizeFractions, setSizeFractions] = useState([])
   const [unit, setUnit] = useState('mD')
-  const [bridgeValueHelperText, setBridgeValueHelperText] = useState(undefined)
-  const [bridgeValueVariant, setBridgeValueVariant] = useState(undefined)
-  const [optimalBridgeGraphData, setOptimalBridgeGraphData] = useState([])
-  const [volumeData, setVolumeData] = useState([])
-  const [cumulativeVolumeData, setCumulativeVolumeData] = useState([])
+  const [bridgeValueHelperText, setBridgeValueHelperText] = useState<string>('')
```

Comment:
```suggestion
  const [bridgeValueHelperText, setBridgeValueHelperText] = useState<string>()
```
Think it is best if this `helperText` either is defined og undefined, not an empty string